apps/authentication/api/connection_token.py

```diff
@@ -670,18 +670,26 @@ def release_applet_account(self, *args, **kwargs):
 
 
 class AdminConnectionTokenViewSet(ConnectionTokenViewSet):
+    serializer_classes = {
+        'default': AdminConnectionTokenSerializer,
+    }
 
     def check_permissions(self, request):
         user = request.user
         if not user.is_superuser:
             self.permission_denied(request)
 
     def get_queryset(self):
-        return AdminConnectionToken.objects.all()
+        return AdminConnectionToken.objects.all().filter(user=self.request.user)
 
     def get_permed_account(self, user, asset, account_name, protocol):
+        """
+        管理员 token 可以访问所有资产的账号
+        """
         with tmp_to_org(asset.org):
-            account = asset.accounts.all().active().get(name=account_name)
+            account = asset.accounts.all().active().filter(name=account_name).first()
+            if not account:
+                return None
             account.actions = ActionChoices.all()
             account.date_expired = timezone.now() + timezone.timedelta(days=365)
             return account
```

Comment:
In the current version, there is no need for changes or suggestions based on the provided code snippet. The code seems to be correct according to knowledge till September 1st, 2021. If you're referring an updated version or have any specific requests regarding changes, feel free to clarify! For instance, please provide further requirements such as whether particular lines should be checked, specific features needed to be added or removed for better functionality (like adding a new class), etc.